index.html

```diff
@@ -4,10 +4,38 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
     <title>Portfolio website </title>
 </head>
 <body>
-    <h1>My portfolio website</h1>
+    <main class="Portfolio-container">
+        <div class="header">
+            <div class="logo">RIDA</div>
+              <div class="menu-container">
+                <img src="Images/icon-menu.png" alt="menu-icon">
+              </div>
+            </div>
```

Comment:
You did great with the visual of this section! 💯 However, there are some changes to consider in your HTML document structure:

- [ ] HTML has a special tag for header sections, the `<header>`. Please prioritize this one over the `<div>` as it doesn't provide any semantics to the browser. 🚧 
- [ ] Your logo, and hamburger menu are supposed to be part of the navigation of your project, the hamburger for your page links later on, and the logo because, by convention, it is used to navigate to the main page of the website ⛵. It makes sense to use a `<nav>` tag to wrap both these elements, as this tag tells the browser that inside it lives the navigation. Also, please change the `div` tag in your logo for an `<a>` tag, and the `div` in your hamburger to either an `<a>` or a `<button>` to describe better what these elements are. 🚧 

```suggestion
        <header class="header">
            <nav>
                <a href="#" class="logo">RIDA</a>
                <button type="button" class="menu-container">
                    <img src="Images/icon-menu.png" alt="menu-icon">
                </button>
            </nav>
        </header>
```

Please consider implementing these changes! 🤓 🛠️